### PR TITLE
Buffs the mind restoration symptom

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

The mind restoration symptom now heals basic brain traumas by default, instead of needing a stat threshold to be met to do so. The resistance 6 threshold effect (instead of the resistance 9 threshold effect) now causes the symptom to heal severe traumas as well, and the resistance 9 threshold now causes the symptom to heal deep-rooted and magic resilience tier traumas.

The mind restoration symptom now activates every tick (instead of every 5-10 ticks). To compensate for this, the chance per activation to cure a trauma has been reduced from 10% to 2%.

I also modified a few descriptions and removed an unneeded empty line (or two? I forgot/forget how many I removed) in the code (the latter shouldn't affect anything gameplay-wise; just the readability of the code).

## Why It's Good For The Game

The mind restoration symptom is kind of weak right now; this PR buffs it so that a virologist can undo the effects of a curse of madness spell or the like if they- wait a minute, curse of madness traumas are of the lobotomy resilience tier, not the magic resilience tier. Well, I know what my next PR is gonna be.

## Changelog
:cl: ATHATH
balance: The mind restoration symptom now heals basic brain traumas by default, instead of needing a stat threshold to be met to do so. The resistance 6 threshold effect (instead of the resistance 9 threshold effect) now causes the symptom to heal severe traumas as well, and the resistance 9 threshold now causes the symptom to heal deep-rooted and magic resilience tier traumas.
balance: The mind restoration symptom now activates every tick (instead of every 5-10 ticks). To compensate for this, the chance per activation to cure a trauma has been reduced from 10% to 2%.
tweak: The mind restoration symptom's description and the descriptions of its threshold effects have been modified to more accurately reflect what they do.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
Fuck you magic man, I have a cold!

**Comment by optimumtact** [comment]
more magic virology bullshit, when will this garbage job be reworked

**Comment by PKPenguin321** [comment]
I don't think magic traumas are even meant to be cured

**Comment by necromanceranne** [comment]
The original PR for Curse of Madness intentionally made them lobotomy tier. Let's not make this already fairly powerful event stronger, okay?

**Comment by ATH1909** [comment]
> I don't think magic traumas are even meant to be cured

Don't staves of healing already heal them?

While yes, this would make viruses the first mundane way to heal magic resilience tier traumas... there are actually very, very few magic resilience tier traumas in the game right now. Tinfoil hats, a 5% chance for a random brain trauma event trauma to be one, traumas from lobotomies... and I guess admin intervention? Those are literally the only sources of TRAUMA_RESILIENCE_MAGIC traumas in the game. I don't think that the world will end if a guy who sits in virology for 15-20 minutes can make a disease that can cure a few types of previously-incurable (by nonmagical means) brain traumas.

File: code/datums/diseases/advance/symptoms/sensory.dm [hunk 1]
```diff
@@ -52,11 +51,13 @@
...
+		if(iscarbon(M))
 			var/mob/living/carbon/C = M
-			if(prob(10))
-				if(trauma_heal_severe)
-					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_LOBOTOMY)
+			if(prob(2))
+				if(trauma_heal_magic)
+					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_MAGIC)  //it might look like we're skipping over a resilience tier (which we technically are), but magic resilience tier traumas are actually really rare, so I lumped them in with the lobotomy resilience tier traumas because I don't think that they're common enough to warrant adding a new threshold effect that just gives this symptom the ability to cure them
+				else if(trauma_heal_severe)
+					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_TRAUMA_RESILIENCE_SURGERY)
```
Comment (XDTM) [inline code/datums/diseases/advance/symptoms/sensory.dm]: TRAUMA_RESILIENCE_TRAUMA_RESILIENCE_SURGERY

File: code/datums/diseases/advance/symptoms/sensory.dm [hunk 2]
```diff
@@ -52,11 +51,13 @@
...
 		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, -3)
-		if(trauma_heal_mild && iscarbon(M))
+		if(iscarbon(M))
 			var/mob/living/carbon/C = M
-			if(prob(10))
-				if(trauma_heal_severe)
-					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_LOBOTOMY)
+			if(prob(2))
+				if(trauma_heal_magic)
+					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_MAGIC)  //it might look like we're skipping over a resilience tier (which we technically are), but magic resilience tier traumas are actually really rare, so I lumped them in with the lobotomy resilience tier traumas because I don't think that they're common enough to warrant adding a new threshold effect that just gives this symptom the ability to cure them
```
Comment (XDTM) [inline code/datums/diseases/advance/symptoms/sensory.dm]: TRAUMA_RESILIENCE_MAGIC is meant for traumas that are effectively permanent barring adminheals or a staff of healing. Think of curing TRAUMA_RESILIENCE_MAGIC as being on the same level as an instant full revive.

**Comment by XDTM** [comment]
Also, curing lobotomy tier traumas automatically without the normal risk of getting a permanent worse trauma is definitely already good here

**Comment by optimumtact** [comment]
We should add a pyschologist job, who has to find a random set of three trigger words of the user by talking to them about their likes and dislikes and then can cure deep seated traumas

**Pull request closed without merging**